src/main/drivers/max7456.c

```diff
@@ -795,13 +636,21 @@ void max7456DrawScreen(void)
         }
 
         if (buff_len) {
-#ifdef MAX7456_DMA_CHANNEL_TX
-            max7456SendDma(spiBuff, NULL, buff_len);
-#else
-            __spiBusTransactionBegin(busdev);
-            spiTransfer(busdev->busdev_u.spi.instance, spiBuff, NULL, buff_len);
-            __spiBusTransactionEnd(busdev);
-#endif // MAX7456_DMA_CHANNEL_TX
+            // This routine doesn't block so need to use static data
+            static busSegment_t segments[] = {
```

Comment:
Betaflight coding convention is to put static variable declarations at the start of the function.

Reply:
Even if they are only used on one code block?